clouddriver-web/config/clouddriver.yml

```diff
@@ -1,12 +1,13 @@
 server:
-  port: 7002
+  port: ${services.clouddriver.port:7002}
+  address: ${services.clouddriver.host:localhost}
```

Comment:
why didn't you remove this from the other five `cloudriver-*.yml` files?

Reply:
Because those are pointing to `service.clouddriver{type}`; while it should always have the same port, the host will be different in general (`spin-clouddriver-ro...`, `spin-clouddriver-rw...`).  So for those cases we do need the overriden value in this hal config...this could probably be cleaned up using profiles, but for now (as we aren't yet doing HA deploys using kleat) I'm just leaving them as is.

Reply:
This hosed us in the upgrade from 1.18->1.19 just fyi. We sat there for over an hour waiting for a clouddriver that would never come :)

Reply:
@clanesf : Can you please provide some more details on how this broke upgrading to 1.19?  (I guess ideally in an issue with details on how you're deploying and any error message, etc.). This should be just moving the same defaults from one file to another to clean things up, though obviously it may have accidentally introduced a bug :) ...but without details it's hard to look into.